### PR TITLE
PT-BR Translation

### PR DESCRIPTION
**Pull request by chakigel**
### PR Creator Checklist

Ensure you've checked the following before submitting your PR:

- [X] You've discussed making your changes with a member of the dev team per contributing rules in the README
- [X] Your changes are free of any lint errors
- [X] Your changes are free of any typescript errors
- [ ] You've tested your changes

### Summary

This adds the PT-BR json file, for the translation as discussed [here](https://beehaw.org/post/5497967)

### Test Plan

> Switch to Portuguese and check the string lengths.

### CODE REVIEW

**Comment by gkasdorf** [comment]
Thank you! I'll merge this in shortly.

File: src/plugins/i18n/locales/pt_br.json [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+      "pushNotificationsDisabled": "Notificações desativadas com sucesso.",
+      "errorFetchingInstanceList": "Erro obtendo a lista de instâncias. Tentar novamente?",
+      "unknownError": "Um erro desconhecido occorreu: Código {{0}}",
+      "sessionExpired": "Sua sessão em {{0}} expirou. Por favor entre novamente nesta conta.",
+      "cacheCleared": "Limpeza de cache concluida.",
+      "accountLogoutConfirm": "Por favor verifique que as notificações foram desativadas da conta antes de sair, caso contrário ainda poderá receber notificações da conta.\n\nTem certeza que quer sair de {{0}}?",
+      "pushNotificationsConfirm": "Para fazer com que as notificações funcionem, Memmy precisa enviar o seu token de autenticação para nosso servidor. Por enquanto esta é a única forma de permitir que os usuários recebam notificações. Importante ressaltar que o servidor do Memmy NÃO tem acesso a sua senha, somente seu token. Tokens de autenticação podem ser revogados mudando sua senha.\n\O código fonte completo do servidor de notificações do Memmy pode ser encontrado em: https:\/\/github.com\/gkasdorf\/memmy-push.\n\nVocê deseja continuar?",
```
Comment (Djones4822) [inline src/plugins/i18n/locales/pt_br.json:42]: Is the `\O` meant to be there?

Comment (Felix-Staud) [reply src/plugins/i18n/locales/pt_br.json:42]: It's a wave

**Comment by Felix-Staud** [comment]
Thanks for your translation.
You are missing adding the language as available inside the `plugins/i18n.ts` file

**Comment by chakigel** [comment]
Hey, thanks for letting me know. 
The two typos @lacarsi mentioned have been fixed and I added the option to `plugins/i18n.ts`.

**Comment by Felix-Staud** [comment]
@gkasdorf @chakigel i think this translation wont work as right now we only use the initial language prefix `pt` not the full one `pt_br` thus the language detector will never provide the proper key for the translation to take effect

**Comment by gkasdorf** [comment]
@Felix-Staud Oh interesting. I'll fix that up then. Thanks.

**Pull request merged**